### DISCUSSION TITLE
Correction of facial detection (newborn baby)

### DISCUSSION THREAD

**Discussion by sucharz** [immich-app/immich#3993] [Q&A] [answered]
Please advise. I have been actively using Immich for some time. I have my photo gallery (>30 000) tagged including faces. Now my 2nd son was born. Immich of course recognizes him as my 1st older son and tags him accordingly. How can I change (correct) this marking? Thank you

**Comment by thomasverelst** [comment, marked as answer]
You can't re-assign faces or split one detection into multiple persons yet. You can try to lower the threshold in the settings so that the algorithm is less likely to group them (Machine Learning > Facial recognition > Max recognition distance). 
However, the face detection algorithm doesn't work as well on newborns and kids as it does on adults, so changing the settings might not help in those situations.
